src/compas_rhino/geometry/surfaces/surface.py

```diff
@@ -37,12 +37,19 @@ class RhinoSurface(Surface):
         True if the surface is periodic in the U direction.
     is_periodic_v: bool
         True if the surface is periodic in the V direction.
+    native_surface: :class:`Rhino.Geometry.Surface`
+        The underlying Rhino surface object.
 
     """
 
     def __init__(self, name=None):
         super(RhinoSurface, self).__init__(name=name)
-        self._rhino_surface = None
+        # empty surface is not really a valid RhinoSurface object.
+        self._rhino_surface = Rhino.Geometry.PlaneSurface(Rhino.Geometry.Plane.WorldXY, Rhino.Geometry.Interval(0, 1), Rhino.Geometry.Interval(0, 1))
```

Comment:
not so sure i like this. how about we add the native surface as an `__init__` parameter instead? potentially with default `None` (since i think using keyword arguments makes multiple inheritance more transparent) and a check of the value of the native surface that raises an errror if it is not a Rhino Surface...

i think the reason this didn't work properly before was because we called `__init__` in the "new plugin" which is not correct, but should be resolved now.

Reply:
from an API point of view, shouldn't there be just one way to do this?
`Surface(native_surface)` makes `from_native` redundant for me. 
I find `Surface.from_native(native_surface)` while a bit verbose, makes it  clear that you need a native CAD surface in order to instantiate a surface object. followin the brep convention.

Looking forward, it also leave the opening to do `Surface()` and not get an error if/when we have our own implementation of it. Same with brep.

Reply:
then i would just leave it as it is. creating an empty surface should not be possible in my opinion. we also don't provide a way to modify a "base" surface, so the only correct way to construct a surface is to use one of the factory functions. i find the implicit creation of planar surface rather opaque behaviour.

Reply:
agree!

Reply:
just to make sure we are talking about the same thing, with "leave it as it is", i mean as it was before the proposed change :)

Reply:
sorry this is taking forever, gonna try and finish this today.

To conclude my understanding and the changes in this PR:

default constructor is not allowed:
```python
from compas.geometry import Surface
Surface()

# this results in
# Runtime error (NotImplementedException): Surface is an abstract class. Use Surface.from_...() to construct a Surface object.
```

```python
surface = Surface.from_native(surface)
type(surface)
# <class 'compas_rhino.geometry.surfaces.surface.RhinoSurface'>
```